app/frontend/src/features/ReportButton.tsx

```diff
@@ -146,8 +150,6 @@ export default function ReportButton({
         aria-labelledby="bug-reporter"
         open={isOpen}
         onClose={handleClose}
-        // won't close when clicking outside
-        disableBackdropClick
```

Comment:
Is this also a deprecated prop? Lol wonder how many things we have to re-learn with their billions of deprecations and changes coming in v5 😅

Reply:
yeah